eng/common/scripts/TypeSpec-Project-Generate.ps1

```diff
@@ -36,17 +108,9 @@ function NpmInstallForProject([string]$workingDirectory) {
             Remove-Item -Path "package-lock.json" -Force
```

Comment:
Don't we also want to remove the places we are deleting the files here? I don't think we want to delete those unless we actually do an npm install.

Reply:
We need to do that in case user installs emitter under the project folder. If we don't do that, node will not load the emitter installed under root. It will load the emitter installed under the projedct folder.

Reply:
If files like package.json are present under this folder do we think that folks have it committed to the repo? If so then deleting it will cause a change in the files. I think we might actually want to warn people instead of blindly deleting these.

Reply:
This place `$workingDirectory` is in the gitignore. It is actually `{root}\sdk\anomalydetector\Azure.AI.AnomalyDetector\TempCadlFiles\AnomalyDetector` for example. 

The scenario I'm thinking is: before this change, there will be `package.json` and `npm_modules` in the `$workingDirectory`. Then users apply this change, and they should load packages from root folder. So I want to remove the  `package.json` and `npm_modules` existing before.

Reply:
If this is under the temp working directory why would there ever be a package.json and node_modules files under that directory?

Reply:
If that is the problem we are worried about we should just have folks remove the entire temp working directory (i.e. `git clean -xdf`). I don't think we should try and special case this one upgrade scenario.